src/containers/task-group.js

```diff
@@ -0,0 +1,90 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import Task from '../components/task';
+
+class TaskGroup extends Component {
+
+  processTaskList(list, givenMonth, givenType) {
+    let resultsArray = [];
+    const loop = list.map( (item) => {
+      if(item.month === givenMonth && item.type === givenType) {
+        resultsArray.push(item);
+      };
+    });
+    if(resultsArray.length) {
+      return true;
+    }
+  }
+
+  taskTypeIsUsed(list, givenMonth, givenType) {
```

Comment:
Similar to above, this method can just be `return list.some(item => this.containsTaskForMonthAndType(item.tasks, givenMonth, givenType)` (with the suggested renaming), aka "return whether any of the items contain a task of given type in the given month".

Also, it might be nice to rename `list` to `itemList` here and `taskList` above. At first glance, I assumed they'd be passing the same list through.